web/package.json

```diff
@@ -13,7 +13,6 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-scripts": "5.0.1",
-    "typescript": "^4.9.5",
     "web-vitals": "^2.1.4"
   },
   "scripts": {
```

Comment:
@sanghyun1128 I don't see any scripts for running `lint` or `format`. Make them as scripts would be better. Please add scripts `lint`, `lint:fix` for eslint, and `format` for prettier.

Reply:
Thx, I added lint, lint:fix, format scripts.